### PR TITLE
fix(demo): fix demo header static position for safari browser

### LINKED ISSUE #3312: fix(demo): demo header is not static for safari browser
### Bug description:
- Open ngx-bootstrap demo at safari browser
- Scroll down any demo page
- See that header scrolls up, and disappears from viewport
### Expected result:
Demo header doesn't scroll up, it should be static

demoheaderissue-2

### PR DESCRIPTION
**Pull request by artolshansky**
close #3312

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated demos.

### CODE REVIEW

File: demo/src/assets/css/style.scss [hunk 1]
```diff
@@ -296,6 +296,7 @@ a:hover {
 }
 /* content */
 .main-container {
+  padding-top: 30px;
```
Comment (EvilAlexei) [inline demo/src/assets/css/style.scss]: Why on .main-container and why 30px?

**Review by EvilAlexei — CHANGES_REQUESTED** [review]

File: demo/src/assets/css/style.scss (hunk 1 above)
Comment (artolshansky) [reply demo/src/assets/css/style.scss]: changed

File: demo/src/assets/css/style.scss [hunk 2]
```diff
@@ -512,9 +512,12 @@ a:hover {
 /*** Media Query ***/
 /* above 768px */
 @media (min-width: 768px) {
+  .main {
```
Comment (EvilAlexei) [inline demo/src/assets/css/style.scss]: Keep an order of elements. Header goes before Main :)

**Review by EvilAlexei — CHANGES_REQUESTED** [review]

**Review by EvilAlexei — APPROVED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Pull request merged**